assets/components/src/plugin-settings/index.js

```diff
@@ -90,8 +90,6 @@ class PluginSettings extends Component {
 			},
 		} )
 			.then( settings => {
-				this.setState( { settings, error: null } );
-
```

Comment:
@miguelpeixe This line is removed because, When user updates more than one settings and did not hit the `save settings` button one after other, then when they change all the settings and then hit any one of the `save settings` button then all the changes he made to settings would get lost because this line fetches the saved settings. Hence, we need to take care of that thing, so instead adding fetched settings we just show settings what are saved in state, which overcomes this issue.

Reply:
Nice catch!